EIPS/eip-4973.md

```diff
@@ -0,0 +1,127 @@
+---
+eip: 4973
+title: Non-Transferrable Non-Fungible Account-bound Tokens (formerly Soulbound Tokens or "Badges")
```

Comment:
```suggestion
title: Non-Transferrable Non-Fungible Account-bound Tokens
```
This name is a bit confusing. I recommend coming up with something more concise.